# Use this file to specify parameters, that do not have default values.
# YAML Files can also be helpful if you want to save parameters for different use-cases.
# In any way: Default values should go into the .if file!
#
# To generate a yaml file, use "rosrun rosinterface_handler generate_yaml /path/to/AnicarDetector.if"
#
# Use 'key: value' pairs, e.g.
# string: 'foo'
# integer: 1234
# float: 1234.5
# boolean: true
# vector: [1.0, 2.0, 3.4]
# map: {"a": "b", "c": "d"}

verbosity: info

color_image_topic: /kinect2/qhd/image_color_rect
depth_image_topic: /kinect2/qhd/image_depth_rect
camera_info_topic: /kinect2/qhd/camera_info
image_publish_topic: /detections_image
pose_publish_topic: /target_pose
points_publish_topic: /detection_points
detect_score_threshold: 0.6
publish_image: false
process_rate: 3
map_frame_id: world
vehicle_frame_id: vehicle_front_axle
